File: tutorial/code/python/pot.py
import zio
import time

agent = zio.Agent("/dev/ttyUSB0")
adc = zio.ADC(agent) # <1>
pot_pin = 0 # <2>

while True:
    value = adc.read_pin(pot_pin) # <3>
    print "ADC Value: %1.2f" % value

    time.sleep(0.5)

File: tutorial/code/java/Pot.java
import com.zilogic.zio.*; // <1>

class Pot {
    public static void main(String args[]) 
	throws ProtocolException, InterruptedException {

	Agent agent = new Agent("/dev/ttyUSB0");
	ADC adc = new ADC(agent); // <1>
	int potPin = 0; // <2>
	
	while (true) {
	    double value = adc.readPin(potPin); // <3>

	    String msg = String.format("ADC Value: %1.2f", value);
	    System.out.println(msg);

	    Thread.sleep(500);
	}
    }
}
